package.json

```diff
@@ -18,117 +18,117 @@
     "access": "public"
   },
   "dependencies": {
-    "@aws-crypto/sha256-js": "^2.0.2",
-    "@aws-sdk/client-sts": "^3.245.0",
-    "@aws-sdk/signature-v4": "^3.226.0",
-    "browserify": "^17.0.0",
-    "clone": "^2.1.2",
-    "cross-fetch": "^3.1.5",
-    "es6-promise": "^4.2.8",
-    "esbuild": "^0.16.16",
-    "exorcist": "^2.0.0",
+    "@aws-crypto/sha256-js": "2.0.2",
+    "@aws-sdk/client-sts": "3.245.0",
+    "@aws-sdk/signature-v4": "3.226.0",
+    "browserify": "17.0.0",
+    "clone": "2.1.2",
+    "cross-fetch": "3.1.5",
+    "es6-promise": "4.2.8",
+    "esbuild": "0.16.16",
+    "exorcist": "2.0.0",
     "express": "4.18.2",
-    "fs-extra": "^11.1.0",
-    "handlebars": "^4.7.7",
-    "jsonpath-plus": "^7.2.0",
-    "mersenne-twister": "^1.1.0",
-    "node-fetch": "^2.6.7",
-    "open": "^8.4.0",
-    "pascalcase": "^1.0.0",
-    "qs": "^6.11.0",
+    "fs-extra": "11.1.0",
+    "handlebars": "4.7.7",
+    "jsonpath-plus": "7.2.0",
+    "mersenne-twister": "1.1.0",
+    "node-fetch": "2.6.7",
+    "open": "8.4.0",
+    "pascalcase": "1.0.0",
+    "qs": "6.11.0",
     "readline-sync": "1.4.10",
     "request": "2.88.2",
-    "request-promise-native": "^1.0.9",
-    "semver": "^7.3.8",
-    "sha.js": "^2.4.11",
+    "request-promise-native": "1.0.9",
+    "semver": "7.3.8",
+    "sha.js": "2.4.11",
     "sinon": "15.0.1",
-    "source-map": "^0.8.0-beta.0",
-    "stacktrace-parser": "^0.1.10",
-    "typescript": "^4.7.4",
-    "uglify-js": "^3.17.4",
-    "url-parse": "^1.5.10",
-    "uuid": "^9.0.0",
-    "xml2js": "^0.4.23",
-    "yargs": "^17.7.1",
-    "yn": "^3.1.1",
+    "source-map": "0.8.0-beta.0",
+    "stacktrace-parser": "0.1.10",
+    "typescript": "4.7.4",
+    "uglify-js": "3.17.4",
+    "url-parse": "1.5.10",
+    "uuid": "9.0.0",
+    "xml2js": "0.4.23",
+    "yargs": "17.7.1",
+    "yn": "3.1.1",
     "zod": "3.20.2"
   },
   "optionalDependencies": {
-    "isolated-vm": "=4.4.1 || >4.4.2"
+    "isolated-vm": "4.4.1"
```

Comment:
We can't put a comment into package.json to remind people not to upgrade to 4.4.2. 4.4.3 isn't released yet, so maybe we just have to collectively remember until 4.4.3 is out

Reply:
Hmm @chris-codaio does Renovate really not have a way to indicate a bad version that you want to ignore? I'd rather not rely on us having to remember this, the bug here was very subtle and I think only affected external developers on another platform and hard to prepare a test for.

Reply:
Ah, yes we can do it in renovate.json

Reply:
https://github.com/coda/packs-sdk/pull/2480